src/common/sha2/sha2_c.c

```diff
@@ -2,51 +2,53 @@
 
 #include <oqs/oqs.h>
 
-#include "sha2.h"
+#include "sha2_local.h"
+#include <stdio.h>
 
 #define PQC_SHA256CTX_BYTES 40
 #define PQC_SHA512CTX_BYTES 72
-
+/*
```

Comment:
Why is this commented out code still here?

Reply:
Good question, I'll remove it!